Jenkinsfile.txt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env groovy
+@Library('lib')_
+ 
+pythonPipeline {
+  package_name = "openeo-cdse-benchmarks"
+  test_module_name = "."
+  wipeout_workspace = true
+  python_version = ["3.6"]
```

Comment:
python 3.6 is end of life, I don't think there is a reason to pick such an old python version for a new project. I think we can go for "3.10" here